api/src/main/java/org/openmrs/hl7/handler/ORUR01Handler.java

```diff
@@ -1203,7 +1202,6 @@ private Date tsToDate(TS ts) throws HL7Exception {
 		int sec = (dtm.length() >= 14 ? Integer.parseInt(dtm.substring(12, 14)) : 0);
 		Calendar cal = Calendar.getInstance();
 		cal.set(year, month, day, hour, min, sec);
-		// if (cal.getTimeZone().getRawOffset() != timeZoneOffsetMillis) {
```

Comment:
@Nuwantha why do you often only remove on line? please remove the entire commented out code section